1001-2000/LIT1799Lifeof.xml

```diff
@@ -1,11 +1,10 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT1799Lifeof">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Life of John the Baptist</title>
-            <!-- apocryphon by Serapion of Thmuis -->
```

Comment:
Serapion is wrong? where is the info now?

Reply:
I removed it. This record was without one single relation to a manuscript. Yes, there is Serapion an alledged author of a Greek version, but is it attested in Ethiopic? I think we did not find it so far.  "The Acts of John by Serapion" would be just misleading a little bit, unless someone can point to a concrete witness of this text. The work which I described is an Ethiopian composition, made of several pieces, unfortunately no sign of Serapion.